.github/workflows/complete-check.yml

```diff
@@ -0,0 +1,19 @@
+name: Complete Check
+
+on:
+  - push
+  - pull_request
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
```

Comment:
We maybe want to explicitly set the NodeJs version?
```
- uses: actions/setup-node@v2
  with:
    node-version: '10'
    check-latest: true
```

Reply:
Can you give an argument of _why_ we would want to specify a Node version, instead of using the latest LTS release (which is the default behavior, currently Node 14)?

Reply:
ok i was wondering which was the default Node version in the runner since i didn't find it in the docs. But if it's already the latest LTS then i guess we don't need this. Unless there was a reason to use Node 10.x with travis.

Reply:
I think it was a left-over from when a package failed to build in a newer version. Seems not relevant anymore, as all tests pass with default version also.